### PR TITLE
feat(manifest): trim manifest with wildcard

### PR DESCRIPTION
**Pull request by clearloop**
Resolves # .

merged the field `members` in the root manifest

- [x] core-backend
- [x] node
- [x] runtime
- [x] utils

@gear-tech/dev

### CODE REVIEW

**Review by tltsutltsu — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -13,63 +13,10 @@ members = [
...
-    "examples/sys-calls",
-    "examples/syscall-error",
-    "examples/vec",
-    "examples/wait",
-    "examples/wait-timeout",
-    "examples/wait_wake",
-    "examples/waiter",
-    "examples/waiting-proxy",
-    "examples/wat",
+    "examples/[!big-wasm]*",
```
Comment (shamilsan) [inline Cargo.toml:19]: We had trouble with wildcards in `examples` some time ago. When you have some new example in a branch, you get a compilation error when returning to the master branch as you have a new example directory (with `.binpath`), but no manifest inside it.

File: Cargo.lock [hunk 2]
```diff
@@ -2053,15 +2053,6 @@ dependencies = [
...
 ]
 
-[[package]]
-name = "demo-ncompose"
-version = "0.1.0"
-dependencies = [
- "gear-wasm-builder",
- "gstd",
- "hex",
-]
```
Comment (shamilsan) [inline Cargo.lock:2063]: Some examples seem to be accidentally removed.

File: Cargo.toml (hunk 1 above)
Comment (clearloop) [reply Cargo.toml:19]: mb we can  `examples/[!big-wasm|!.binpath]` when meet that problem?

File: Cargo.lock (hunk 2 above)
Comment (clearloop) [reply Cargo.lock:2063]: yeah just fount it is pretty buggy, considering creating another issue for the examples

Comment (clearloop) [reply Cargo.lock]: https://github.com/gear-tech/gear/issues/3069

File: Cargo.toml [hunk 3]
```diff
@@ -83,29 +80,11 @@ members = [
     "gstd",
     "gsys",
     "gtest",
-    "node/cli",
-    "node/service",
-    "node/authorship",
-    "node/testing",
+    "node/[!res]*[!md]",
```
Comment (shamilsan) [inline Cargo.toml:83]: I'd leave the old list for `node` as this `[!res]*[!md]` looks not too cool to me.

**Review by shamilsan — APPROVED** [review]

File: Cargo.toml (hunk 3 above)
Comment (clearloop) [reply Cargo.toml:83]: but this is a one-line solution )) mb create a poll : D

Comment (clearloop) [reply Cargo.toml]: leave it to #3069

**Pull request merged**